File: test-crypto.js
const crypto = require('crypto');
const fs = require('fs');

const privKey = fs.readFileSync('priv.pem');
const data = fs.readFileSync('requirements.txt');

const signer = crypto.createSign('RSA-SHA256');
signer.write(data);
signer.end();

const signature = signer.sign(privKey, 'base64');

console.log(signature);

File: test-crypto.py
from Crypto.Signature import pkcs1_15
from Crypto.Hash import SHA256
from Crypto.PublicKey import RSA
import base64

public_key = RSA.import_key(open('pub.pem', 'rb').read())
data = open('requirements.txt', 'rb').read()

signature = base64.b64decode(open('signature.txt', 'rb').read())

pkcs1_15.new(public_key).verify(SHA256.new(data), signature)
